packages/Makefile

```diff
@@ -1,12 +1,13 @@
 
-YB_VERSION ?= 2.1.1.0
+YB_VERSION ?= 2.1.2.0
+YB_CLIENT_VERSION ?= 2.1.1.0
```

Comment:
We changed this recently to be just the first two digits: 2.1 instead of 2.1.1.0.

Reply:
As per our discussion, we can keep the version as 2.1 and update the package revision whenever there is a new release for client package in that case. Should I change this version to 2.1 instead of 2.1.1.0?